front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -6,8 +6,10 @@ import Manchette, {
   type ProjectPathTrainResult,
   type WaypointMenuData,
 } from '@osrd-project/ui-manchette';
+// TODO: export InteractiveWaypoint type from ui-manchette
```

Comment:
Note, this is addressed in https://github.com/OpenRailAssociation/osrd-ui/pull/942